### DISCUSSION TITLE
How does a player choose their move when in a battle with another user ?

### DISCUSSION THREAD

**Discussion by shinegami-2002** [brianhhuynh38/slackpoint-v3#21] [General]
While implementing the /attack command, there is one thing which we are forgetting, how would a person choose which move they want to use in a attack.

**Comment by brianhhuynh38** [comment]
Given how many of the commands use content input forms in order to obtain information from the player, we can use the same layout here. A rich text block can be used to display all pertinent information to the player regarding their class move set, then they can choose their move via a dropdown, or some kind of input block that can parse the player input.

```
Opponent HP: 90/300

HP: 300/300
MP: 20/100

Your turn!

1) Slash (20 power, 80 hit)                     0 MP
2) Heavy Slash (60 power, 90 hit)               25 HP
3) ...

____________________________________
[___________________________________] Input field of some sort
```

**Reply by shinegami-2002** [reply]
added the functionality in battle_commands, where the player has the ability to select their move
